shared/src/lib.rs

```diff
@@ -31,3 +37,15 @@ pub mod transport;
 pub mod web3_traits;
 
 pub type Web3 = web3::Web3<transport::MetricTransport<crate::http_transport::HttpTransport>>;
+
```

Comment:
We can also have a `types` module or something for the `H160Wrapper` instead of putting it in the root - but no strong feelings.